src/pages/List.js

```diff
@@ -16,18 +16,14 @@ export default function List({ token }) {
   }
 
   const markItemPurchased = (e, id, itemData) => {
-    const currentTimestamp = Date.now();
-    const initialInterval = itemData.purchase_frequency * 86400000;
+    const currentTimestamp = Math.floor(Date.now() / 86400000);
```

Comment:
It's unclear to me what this means. I'm noticing your `last_purchased` values in Firestore to be really close in number to others.
<img width="665" alt="Screen Shot 2021-05-12 at 1 26 42 PM" src="https://user-images.githubusercontent.com/6759658/118040424-56b89a80-b326-11eb-80c8-effa8713ae8f.png">

Reply:
Has this been resolved @jssckbl @jamesncox? If so, please resolve the conversation to keep the PR clean.

Reply:
Hey Sky, sorry I didn't see your comment here. Yes, this has been resolved!